hkmc2/shared/src/test/mlscript/handlers/HandlerBlockBidnings.mls

```diff
@@ -0,0 +1,34 @@
+:js
+:handler
+
+
+abstract class Effect with
+  fun perform(arg: Str): Str
+
+
+handle h = Effect with
+  fun perform(arg)(k) =
+    print(arg)
+let x = 123
+
+:fixme
+x
+//│ ╔══[ERROR] Name not found: x
+//│ ║  l.15: 	x
+//│ ╙──      	^
```

Comment:
the rationale of this is that `x` may not be initialized if the handler is non-resumptive

Reply:
Ah, good point... For now it's fine like this, then. But later when we have optional bindings, we can mark it as optional.